terraform/development/main.tf

```diff
@@ -36,3 +36,9 @@ terraform {
     key     = "services/person-api/state"
   }
 }
+
+resource "aws_sns_topic" "user_updates" {
+  name                        = "Person"
```

Comment:
@evak6979 For FIFO topics, the name should end with ".fifo" I believe

Reply:
I thought so too but didn't see it as part of the ticket.